platform/ios/export/export_plugin.cpp

```diff
@@ -2528,11 +2566,59 @@ Error EditorExportPlatformIOS::_export_project_helper(const Ref<EditorExportPres
 	if (ep.step("Making .xcarchive", 3)) {
 		return ERR_SKIP;
 	}
+	
+	if (p_preset->get("application/enable_cocoapods").operator bool()) {
+		// Gather CocoaPods dependencies
+		Dictionary dependencies;
+		for (int i = 0; i < assets.size(); ++i) {
+			const IOSExportAsset &asset = assets[i];
+			if (!asset.is_pod)
+				continue;
+			
+			Vector<String> parts = asset.exported_path.split(":");
+			String name = parts[0];
+			String version = parts[1];
+			
+			if (dependencies.has(name) && dependencies[name] != version) {
+				add_message(EXPORT_MESSAGE_ERROR, TTR("CocoaPods Setup"), vformat(TTR("Ambiguous CocoaPod dependency, %s wants version %s and %s"), name, version, dependencies[name]));
+				return err;
+			}
+			
+			dependencies[name] = version;
+			print_line("Added " + name + " : " + version + " to cocoapods dependencies");
+		}
+		
+		// Generate the podfile
+		err = _generate_podfile("ios", p_preset->get("application/min_ios_version").operator String(), binary_name, dependencies, dest_dir);
+		if (err != OK) {
+			add_message(EXPORT_MESSAGE_ERROR, TTR("CocoaPods Setup"), vformat(TTR("Failed to generate Podfile with code %d"), err));
+			return err;
+		}
+		
+		// Install the pods
+		List<String> pod_args;
+		pod_args.push_back("install");
+		pod_args.push_back("--project-directory=" + dest_dir);
+		
+		String pod_str;
+		OS::get_singleton()->set_environment("LANG", "en_US.UTF-8"); // Required for Cocoapods to function
+		err = OS::get_singleton()->execute("/usr/local/bin/pod", pod_args, &pod_str, nullptr, true);
```

Comment:
I wasn't able to just use `"pod"` as the command to execute here, what would be a better way to execute the right process? I can imagine this path not being 100% the same on all systems